src/turn_attachments.py

```diff
@@ -0,0 +1,77 @@
+import requests
+import mimetypes
+import argparse
+import utils
+from pathlib import Path
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Sync WhatsApp attachments to GCS and notify on errors.")
+    parser.add_argument(
+        '--params-path',
+        type=Path,
+        default=Path('params.yaml'),
+        help='Path to the params.yaml file'
+    )
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    params = utils.get_params(args.params_path)
+
+    try:
+        # Fetch recent attachments from BigQuery
+        project_analytics = params['project_analytics']
+        creds_analytics = params['credentials_analytics']
+        query = f"""
+          SELECT *
+          FROM `{project_analytics}.prod.res_message_attachments`
+          WHERE inserted_at >= TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 DAY)
+            AND media_type NOT IN ('location', 'sticker')
+          ORDER BY channel_phone
+          """
+        df = utils.run_read_bigquery(query, creds_analytics)
+
+        # Prepare GCS bucket
+        storage_client = utils.get_storage_client(params['project_raw'], params['credentials_raw'])
+        bucket = storage_client.bucket(params['bucket_name'])
+
+        # Download each attachment and upload to GCS
+        for _, row in df[1:4].iterrows():
+            uri = row['uri']
+            media_type = row['media_type']
+            mime_type = row['mime_type']
+            phone = row['channel_phone']
+            headers = params['turn_headers'].get(phone)
+            
+            if headers is None:
+                #print(f"No headers configured for {phone!r}, skipping {uri}")
+                continue
+
+            response = requests.get(uri, headers=headers)
+            if response.status_code != 200:
```

Comment:
for clarity, would consider reversing the logic of the `if` and putting the code below inside the `if`.